app/models/post.rb

```diff
@@ -1,6 +1,6 @@
 class Post < ApplicationRecord
-  belongs_to :author, foreign_key: :user_id
-  belongs_to :receiver, foreign_key: :receiver_id
+  belongs_to :author, foreign_key: :author_id, :class_name => "User"
```

Comment:
This was one of those funny mismatch things. We were using author and user_id. Rails error messages would use authors as a resource, but the schema would use `user_id`. We should stick with one, I went with `author` and `author_id`.